README.md

```diff
@@ -38,27 +38,21 @@ For additional examples, browse our complete [API documentation](https://docs.ra
 
 | Algorithm | Scale | Notes |
 | --- | --- | --- |
-| Truncated Singular Value Decomposition (tSVD) | Single GPU | Multi-GPU available in conda cuda10 package |
-| Linear Regression (OLS) | Single GPU | Multi-GPU available in conda cuda10 package <br> Multi-Node with [dask-cuml](http://github.com/rapidsai/dask-cuml) |
-| Principal Component Analysis (PCA) | Single GPU |
-| Density-Based Spatial Clustering of Applications with Noise (DBSCAN) | Single GPU |
-| K-Means Clustering | Multi-GPU |
-| K-Nearest Neighbors (KNN) | Multi-GPU | Multi-Node with [dask-cuml](http://github.com/rapidsai/dask-cuml) <br> Uses [Faiss](https://github.com/facebookresearch/faiss) |
+| Linear Regression (OLS) | Single GPU | Multi-GPU available in conda cuda10 package and [dask-cuml](http://github.com/rapidsai/dask-cuml) |
+| Stochastic Gradient Descent | Single-GPU | for linear regression, logistic regression, and linear svm with L1, L2, and elastic-net penalties |
+| Coordinate Descent | Single-GPU | |
 | Ridge Regression | Single-GPU |
-| Kalman Filter | Single-GPU |
 | UMAP | Single-GPU |
-| Stochastic Gradient Descent | Single-GPU | for linear regression, logistic regression, and linear svm with L1, L2, and elastic-net penalties |
+| Density-Based Spatial Clustering of Applications with Noise (DBSCAN) | Single GPU |
+| K-Means Clustering | Single-GPU |
+| K-Nearest Neighbors (KNN) | Multi-GPU with [dask-cuml](http://github.com/rapidsai/dask-cuml) <br> Uses [Faiss](https://github.com/facebookresearch/faiss) |
+| Principal Component Analysis (PCA) | Single GPU |
+| Truncated Singular Value Decomposition (tSVD) | Single GPU | Multi-GPU available in conda cuda10 package |
+| Linear Kalman Filter | Single-GPU |
 
 ---
 
-Algorithms in progress:
-
-- More Kalman Filter versions
-- Lasso
-- Elastic-Net
-- Logistic Regression
-
-More ML algorithms in cuML and more ML primitives in ml-prims are being worked on. Goals for future versions include more algorithms and multi-gpu versions of the algorithms and primitives.
+More ML algorithms in cuML and more ML primitives in ml-prims are being worked on, among them t-sne, decision trees, random forests and others. Goals for future versions include more multi-gpu versions of the algorithms and primitives.
```

Comment:
Josh left a comment on our planning spreadsheet that he didn't want us pushing decision trees since we have XGBoost. For this reason, I removed the decision trees line from that doc. We should mention, at the very least, spectral embedding, spectral clustering, random projections, SVM, and probability prims.